src/js/view/controls/components/volumetooltipicon.ts

```diff
@@ -81,14 +81,21 @@ export default class VolumeTooltipIcon extends TooltipIcon {
         this.addSliderHandlers(this.horizontalSlider.uiOver);
         this.addSliderHandlers(this.verticalSlider.uiOver);
 
-        this.onAudioMode(null, _model.get('audioMode'));
+        this.updateSlider(_model, _model.get('audioMode'));
 
-        this._model.on('change:audioMode', this.onAudioMode, this);
+        this._model.on('change:audioMode', this.updateSlider, this);
```

Comment:
You can replace both lines above with:

```
this._model.change('audioMode', this.updateSlider, this);
```

`model.change` adds the attribute change callback and invokes it with the current model attribute value all at once.